### DISCUSSION TITLE
Tutorial 1 not outputting pressure values in exodus file

### DISCUSSION THREAD

**Discussion by diegoaspinwall** [idaholab/moose#27764] [Q&A Getting Started] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (avoid screenshots if possible, triple back quotes before/after pasted text, etc)

### Issue or question about MOOSE

Hi everyone,

I recently installed moose on my machine. I'm running Windows 11, so I installed WSL and Ubuntu 20.04, as instructed. Then I git cloned the library this morning and have been working to reproduce the [first tutorial](https://mooseframework.inl.gov/getting_started/examples_and_tutorials/tutorial01_app_development/step01_moose_app.html), babbler.

I'm following the first tutorial exactly so far, except I never linked my code to github, which I imagine wouldn't affect anything. I made the pressure_diffusion.i file exactly the same as in the tutorial as well. I then run

```
cd ~/projects/babbler/problems
../babbler-opt -i pressure_diffusion.i
``` 

and get my 'pressure_diffusion_out.e' file. But when I plot it in ParaView, all pressure values are zero.

The weird thing is that I can run the [first example](https://mooseframework.inl.gov/getting_started/examples_and_tutorials/examples/ex01_inputfile.html), and I get a correct diffusion gradient.

Not sure what is going on. Does anyone have ideas I could try? Thank you.

### Diagnostics for MOOSE installation issues

[Optional] Output of the diagnostics scripts. To run the script: cd ~/projects/moose/scripts; ./diagnostics.sh

**Comment by GiudGiud** [comment, marked as answer]
Hello

Did you make sure to visualize the second time step and not the initial condition?

Guillaume

**Reply by diegoaspinwall** [reply]
Hi Guillaume - thank you so much. I just did that and it worked. I will not be making that mistake again lol
